app/models/thredded/messageboard_group.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module Thredded
+  class MessageboardGroup < ActiveRecord::Base
+    has_many :messageboards
```

Comment:
If a category is destroyed, the foreign key should be nullified, so `dependent: :nullify`. Also, `inverse_of: :messageboard_group`.